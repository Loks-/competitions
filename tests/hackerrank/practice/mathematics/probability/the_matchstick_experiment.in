2
2 2 0.50
2 3 0.75
