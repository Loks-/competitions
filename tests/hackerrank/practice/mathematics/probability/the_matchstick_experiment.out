0.4375000000
0.0810546875
